<?xml version="1.0" encoding="UTF-8"?>
<launch>
  <arg name="follow" default="no"/>

  <param name="mdelay" type="int" value="75" />
  <param name="duty_cycle" type="int" value="50" />
  <param name="target" type="int" value="0" />
  <param name="cc" type="bool" value="True" />

  <node name="client" pkg="sphere_control" type="joystick_control_client.py" output="screen"/>
  <group if="$(eval arg('follow') == 'yes')">
    <node name="homing" pkg="sphere_control" type="home_in.py" output="screen"/>
  </group>
  <!--node name="gui" pkg="sphere_control" type="sphere_gui.py" output="screen"/-->
</launch>